.jenkins/static.groovy

```diff
@@ -13,16 +13,16 @@ def runCI =
 {
     nodeDetails, jobName->
 
-    def prj  = new rocProject('hipBLAS', 'StaticLibrary')
-    prj.paths.build_command = './install.sh -cd --static -p /opt/rocm/lib/cmake'
+    def prj  = new rocProject('hipBLAS', 'static')
+    prj.paths.build_command = './install.sh -cs --compiler=hipcc'
```

Comment:
So is hipcc required?

Reply:
Yes, it seems so
When using g++, the job would fail in the compile stage: http://math-ci.amd.com/blue/organizations/jenkins/static-libraries%2Fstatic%2FhipBLAS/detail/PR-795/15/pipeline/452